design-documents/graph-ql/coverage/dynamic-blocks.graphqls

```diff
@@ -0,0 +1,49 @@
+type DynamicBlock {
+    uid: ID!
+    name: String!
+    content: String!
```

Comment:
We can make it an object, that would allow to return content as a JSON in the future.